bucket/crunchyroll-go.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "2.1.0",
+    "description": "A Go library & CLI for the undocumented Crunchyroll API.",
+    "homepage": "https://github.com/ByteDream/crunchyroll-go",
+    "license": "LGPL-3.0 License",

```

```suggestion
    "license": "LGPL-3.0-only",
```

License should be a valid SPDX identifier. https://spdx.org/licenses/

---

thought I made that change already, must have accidently removed it.